portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +30,41 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Himani!");
+
+    // Create a list of hard-coded comments.
+    List<Comment> comments = new ArrayList<>();
+
+    String name1 = "Himani Yadav";
+    String message1 = "Wow, this is a great website :)";
+    Comment comment1 = new Comment(name1, message1);
+
+    String name2 = "Jon Snow";
+    String message2 = "I know nothing :)";
+    Comment comment2 = new Comment(name2, message2);
+    
+    String name3 = "Arya Stark";
+    String message3 = "Winter is coming.";
+    Comment comment3 = new Comment(name3, message3);
+
+    comments.add(comment1);
+    comments.add(comment2);
+    comments.add(comment3);
+
+    // Convert the list of comments to JSON
+    String json = convertToJson(comments);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+  }
+
+  /**
+   * Converts a Comment instance into a JSON string using the Gson library.
+   */
+  private String convertToJson(List comments) {
+    Gson gson = new Gson();
+    String json = gson.toJson(comments);
+    return json;
```

Comment:
Since the gson object is only used to convert the list to json, it's not really necessary to create a variable for it as it takes up unnecessary space. Similar idea applies to the json String. Instead we can change this to a 1 line return statement:
```suggestion
    return new Gson().toJson(comments);
```